Gemfile

```diff
@@ -1,7 +1,6 @@
 source "https://rubygems.org"
 
+gem "kitchen-docker", ">= 2.9"
+gem "kitchen-salt", ">=0.6.0"
 gem "test-kitchen", '>=1.23.2'
-gem "kitchen-docker"
-gem "kitchen-salt", ">=0.2.5"
 gem "kitchen-inspec"
-
```

Comment:
Since we're changing `Gemfile`, can we strip it down to this:

```ruby
gem 'kitchen-docker', '>= 2.9'
gem 'kitchen-salt', '>= 0.6.0'
gem 'kitchen-inspec', '>= 1.1'
```

`test-kitchen` will be pulled in via. dependency resolution.

Reply:
good point changed it.

Reply:
That's `gem 'kitchen-inspec', '>= 1.1'` instead of `gem "test-kitchen", '>= 1.1'`.

Reply:
oops